src/vendor/wordpress/_wordpress.scss

```diff
@@ -55,6 +83,40 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Embed
+ * Applies our pattern library styles to gutenberg embeded blocks
+ */
+
+.wp-block-embed {
+  &.o-embed--square .wp-block-embed__wrapper {
+    --aspect-ratio: #{aspect-ratio.$square};
+  }
+
+  &.o-embed--full .wp-block-embed__wrapper {
+    --aspect-ratio: #{aspect-ratio.$full};
+  }
+
+  &.o-embed--cinema .wp-block-embed__wrapper {
+    --aspect-ratio: #{aspect-ratio.$cinema};
+  }
+
+  &.alignleft {
+    float: left;
+    margin-right: #{sizes.$rhythm-space};
```

Comment:
We should create a new design token for horizontal gaps, `rhythm-space` is intended for the vertical rhythm object.

Reply:
I added some new design tokens in 7233962 and I'd appreciate some guidance on the appropriateness of the names and values I selected.